src/web/browser/ophan/ophan.ts

```diff
@@ -54,7 +54,8 @@ export type OphanComponentType =
     | 'ACQUISITIONS_EDITORIAL_LINK'
     | 'ACQUISITIONS_SUBSCRIPTIONS_BANNER'
     | 'ACQUISITIONS_OTHER'
-    | 'SIGN_IN_GATE';
+    | 'SIGN_IN_GATE'
+    | 'BRAZE_BANNER';
```

Comment:
Is creating a new component type as simple as defining it here? Does it need to be added somewhere else first?

Reply:
You might need to add this to the Thrift model, if you do you then have to release it to 5+ other apps so I'd like to add another one in there for Contributions if you do. cc @tomrf1

Reply:
The enum in the ophan model will need to be updated before it can receive these events, but they probably do not need to release the thrift model to maven (unless something else depends on that)

Reply:
Yep - and note that within [the Ophan stack, the Slab and everything downstream from it](https://github.com/guardian/ophan/blob/master/docs/how-ophan-works/ophan-stack.md) will need to be deployed with the update before Ophan will recognise the new value.

Reply:
This is good to know, thanks all :+1: